L1Trigger/L1TTrackMatch/plugins/L1TkHTMissEmulatorProducer.cc

```diff
@@ -204,6 +206,8 @@ void L1TkHTMissEmulatorProducer::produce(edm::Event& iEvent, const edm::EventSet
       continue;
     if (tmp_jet_nt < minNtracksHighPt_ && tmp_jet_pt > 400)
       continue;
+    if (displaced_ && !IsDisplaced)
+      continue;
```

Comment:
I'm not sure I get the logic here? Is this checking if the jet is both displaced and not displaced? Why would this check ever be true?

Reply:
The "IsDisplaced" flag comes from each jet in line 144. the "displaced_" is set by the user when we run on displaced jets line 67. The value will be true if the jet is not displaced but we run on displaced "mode".

Reply:
https://github.com/cms-l1t-offline/cmssw/blob/1c7e0cade25535e13c541186d6914845be44c0ba/L1Trigger/L1TTrackMatch/python/l1tTrackerEmuHTMiss_cfi.py#L22

Reply:
It's not a strict issue, but this is going to run the risk of being very confusing to future users and maintainers of this code.

Reply:
```IsDisplaced``` could be ```JetIsDisplaced``` ?

Reply:
to be very strict, it could be at least of the same format as the other "temp" jet variables in https://github.com/cms-l1t-offline/cmssw/blob/1c7e0cade25535e13c541186d6914845be44c0ba/L1Trigger/L1TTrackMatch/plugins/L1TkHTMissEmulatorProducer.cc#L139-L142

i.e. `tmp_jet_dispFlag_ = jetIter->dispflag();`

Reply:
Hi all, thanks I renamed the variable. Since we need the HT for the trigger review if you think that is ok, can we merge it ?

Reply:
@gkaratha Sure. Let me just rerun the tests and then we can merge this.

Reply:
@aloeliger thanks a lot!!!